BearlyHumans/squirrelville
# Updated intro cutscene

Steps to test:
1. Play game
2. Watch intro cutscene
3. Make sure it's the same version as [4nov_final_cutscene_with_audio.mp4](https://drive.google.com/file/d/1yybw9nxcqozt3Ybowa07ZCA7W7Lwiqv3/view?usp=sharing)
